index.html

```diff
@@ -21,12 +21,27 @@
   </head>
   <body>
     <header>
-      <a href="index.html" class="primary nav-header">My Logo</a>
+      <div id="myNav" class="overlay">
+        <a class="closebtn" id="closebtn">&times;</a>
+        <div class="overlay-content" id="overlay-content">
+          <a href="#Portfolio">Portfolio</a>
+          <a href="#About">About</a>
+          <a href="#Contact">Contact</a>
+        </div>
+      </div>
+      <a href="index.html" id="nav-header" class="primary nav-header"
+        >My Logo</a
+      >
       <nav>
-        <img class="nav-img" src="./images/menu.svg" alt="navigation menu" />
-        <a href="#"> Portfolio </a>
-        <a href="#"> About </a>
-        <a href="#"> Contact </a>
+        <img
+          id="img"
+          class="nav-img"
+          src="./images/menu.svg"
+          alt="navigation menu"
+        />
+        <a href="#Portfolio"> Portfolio </a>
+        <a href="#About"> About </a>
+        <a href="#Contact"> Contact </a>
       </nav>
```

Comment:
- [ ] You did a good job on the project functionalities but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design as much as possible. Please decrease the size of mobile menu options and the close(X) button and make sure it matches the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/174995445-c298d155-1f95-469f-aad4-05e9c3c3b5a4.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/174995496-673b434f-3337-4702-94aa-ebe9a37858c1.png)